src/components/grid-aware/StatsBarBlock/StatsBarBlock.module.css

```diff
@@ -0,0 +1,218 @@
+.gray {
+  background-color: var(--color-gray-300);
+}
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: var(--bleed-grid-template-columns);
+  margin: auto;
+  max-width: var(--bleed-max-width);
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.bleedWrapper > * {
+  min-width: 0;
+}
+
+.bleedBackground {
+  grid-column: 2 / span 2;
+  grid-row: 1;
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr 40px;
+  }
+}
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  padding: 100px 0 150px;
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+    padding: 50px 0 38px;
+  }
+}
+
+.gridAreaBottom {
+  display: flex;
+  flex-direction: row;
+  grid-column: 2 / span 10;
+  grid-row: 3;
+  margin-top: 54px;
```

Comment:
I'm seeing a margin of 50px in the Figma design:

<img width="496" alt="Screen Shot 2022-03-16 at 11 15 37 PM" src="https://user-images.githubusercontent.com/1002748/158748379-1d790a7e-38c1-42ac-bdad-cdd9d2a28d13.png">

Could you change this to 50px?

```suggestion
  margin-top: 50px;
```